### ISSUE TITLE
Progress color prop is ignoring non-Bootstrap colors

### ISSUE THREAD

**Issue by fmahmud** [facultyai/dash-bootstrap-components#819]
I looked and couldn't find any mention of this. Hopefully, this isn't my misunderstanding! Great work on the project so far and thank you!

In the documentation it is claimed that the color prop for the Progress component supports hex codes or CSS color names:
```
color (string; optional): Set color of the progress bar, options: primary, secondary, success, warning, danger, info or any valid CSS color of your choice (e.g. a hex code, a decimal code or a CSS color name).
```

It can be seen at this line of code, 
https://github.com/facultyai/dash-bootstrap-components/blob/main/src/components/progress/Progress.js#L33 

(at the time of writing:`variant={isBootstrapColor ? color : null}` ) - we are ignoring any color value that isn't in the "primary", "secondary"... etc. format. 

Either we should change the documentation or we should update the code. 

Edit: This is probably the PR that introduced the problem: https://github.com/facultyai/dash-bootstrap-components/pull/646/commits/3412ccae5ce52baff4496197f4797dc077351fe4#diff-e5b2cb6a3417c85daae955adf86f139ac808ba9654979a88c8206fe5190485b9L40
because it was definitely working before.

**Comment by tcbegley** [comment]
Hi @fmahmud 

Thanks for raising this and the investigative work! Yes, it seems very likely that the migration to React-Bootstrap for v1 allowed this bug to creep in.

I don't see why it wouldn't be possible to fix for the next release.

**Comment by fmahmud** [comment]
Thank you so much! Looking forward to using this feature again soon :)

**Comment by tcbegley** [comment]
Hey @fmahmud 

Sorry for the delay, this has been fixed in #835 and is available in a beta for the next release. If you have a moment to try it out I'll be very know that it works for you.

```
pip install dash-bootstrap-components==1.1.0b1
```

**Comment by tcbegley** [comment]
This is now available in the latest release

```
pip install -U dash-bootstrap-components
```
